### PR TITLE
Add convnet model for benchmark.

### PR DESCRIPTION
**Pull request by qingqing01**
The usage is:

```bash
python benchmark_convnet_model.py --batch_size 32 --iterations 50
```

The `--use_cprof`, `--use_nvprof` is off by default.

### CODE REVIEW

File: paddle/benchmark_convnet_model.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    duration = time.time() - start_time
+    examples_per_sec = args.iterations * args.batch_size / duration
+    sec_per_batch = duration / args.batch_size
+
+    print('\nTotal examples: %d, total time: %.5f' %
+          (args.iterations * args.batch_size, duration))
+    print('%.5f examples/sec, %.5f sec/batch \n' %
+          (examples_per_sec, sec_per_batch))
+
+    if args.use_cprof:
```
Comment (dzhwinter) [inline paddle/benchmark_convnet_model.py]: Maybe it will be better also write stats to file.

File: paddle/benchmark_convnet_model.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        '--iterations', type=int, default=100000, help='The number of minibatches.')
+    parser.add_argument(
+        '--pass_num', type=int, default=100, help='The number of minibatches.')
+    parser.add_argument(
+        '--order',
+        type=str,
+        default='NCHW',
+        help='The data order: (NCHW, NHWC).')
+    parser.add_argument(
+        '--device', type=str, default='GPU', help='The data order: (GPU, CPU).')
```
Comment (chengduoZH) [inline paddle/benchmark_convnet_model.py]: You can use "[choices](https://docs.python.org/3/library/argparse.html#choices)" in parser.add_argument.

File: paddle/benchmark_convnet_model.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+    train_reader = paddle.batch(
+        paddle.reader.shuffle(
+            paddle.dataset.flowers.train(), buf_size=5120),
+        batch_size=args.batch_size)
+
+    if args.device == 'CPU':
+        place = fluid.CPUPlace()
+    else:
+        place = fluid.GPUPlace(0)
```
Comment (chengduoZH) [inline paddle/benchmark_convnet_model.py]: `place = fluid.CPUPlace() if fluid.CPUPlace() else fluid.GPUPlace(0)`

File: paddle/benchmark_convnet_model.py [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+        '--batch_size', type=int, default=32, help='The minibatch size.')
+    parser.add_argument(
+        '--iterations', type=int, default=100000, help='The number of minibatches.')
+    parser.add_argument(
+        '--pass_num', type=int, default=100, help='The number of minibatches.')
+    parser.add_argument(
+        '--order',
+        type=str,
+        default='NCHW',
+        help='The data order: (NCHW, NHWC).')
```
[the remark this answers is not in the document]
Comment (qingqing01) [reply paddle/benchmark_convnet_model.py]: Done.

File: paddle/benchmark_convnet_model.py (hunk 2 above)
Comment (qingqing01) [reply paddle/benchmark_convnet_model.py]: Done.

File: paddle/benchmark_convnet_model.py (hunk 3 above)
Comment (qingqing01) [reply paddle/benchmark_convnet_model.py]: Done.

File: paddle/benchmark_convnet_model.py (hunk 1 above)
Comment (qingqing01) [reply paddle/benchmark_convnet_model.py]: If need, we can add it later.

**Pull request merged**